docs/dev/table/sql.md

```diff
@@ -1510,6 +1510,17 @@ ATAN(numeric)
       </td>
     </tr>
 
+    <tr>
+      <td>
+        {% highlight text %}
+ATAN2(numeric)
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Calculates the arc tangent of a given coordinates.</p>
```

Comment:
should be `of a given coordinate.` (-s)?

Reply:
I checked the [wiki](https://en.wikipedia.org/wiki/Atan2) about atan2, it said: The atan2 function calculates one unique arc tangent value from two variables y and x. So, would it be better `of two variables`?  atan2(y,x) can be the arc tangent of (x,y) or (nx, ny).